Readme.md

```diff
@@ -0,0 +1,136 @@
+## What kind of syslog-ng docker images are exist
+
+  * There are two types of docker images:
+     * balabit/syslog-ng image where syslog-ng is installed in a fresh Debian 8 (Jessie) operation system from (official) deb package. This docker image is for to use syslog-ng as system logging service.
+     * balabit/syslog-ng-dev development image. This image based on also a fresh Debian 8 (Jessie). This image is not contain syslog-ng, it only contains all packages to build and debug syslog-ng within the container.
```

Comment:
I'd remove `from (official) deb package`. I don't think the users are interested in it and the `official` word maybe a little strong.